src/app/body/milestone-details/milestone-details.component.html

```diff
@@ -51,11 +51,12 @@
       <div class="col-md-4">
         <div class="card p-3">
           <div class="row py-3">
-            <div class="col text-center">
+            <div class="col-11 text-center">
               <span class="title">
                 {{milestoneData.Title}}
               </span>
             </div>
+            <div id="milColor" [ngStyle]="{'background-color': color}"></div>
```

Comment:
Give this div class of col

Reply:
sir class col to this div disturb ui of card without that class it works fine and responsive too